public/css/list/redundancy-group-list-item.less

```diff
@@ -0,0 +1,21 @@
+.redundancy-group-list-item.list-item {
+  .caption {
+    display: flex;
+    justify-content: space-between;
+
+    .plugin-output {
+      vertical-align: middle;
+      .text-ellipsis();
+    }
```

Comment:
Hmm... shouldn't this be handled already because it's a list item? Either way, it should not behave differently than output shown as part of a host or service.

Reply:
`display: flex` rule added to the caption actually breaks `text-overflow: ellipsis`. Hence, I added it here again.

Reply:
I have just noticed that I have to also clamp the lines down to 2 to replicate the common view mode.